docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -15,6 +15,7 @@ with available Gradle, Xcode, and Android Gradle plugin versions:
 
 | Kotlin version | Gradle    | Android Gradle plugin | Xcode |
 |----------------|-----------|-----------------------|-------|
+| 2.0.21         | 7.5-8.8*  | 7.4.2–8.5             | 16.0  |
```

Comment:
Well, technically 2.0.21 isn't out yet.  
Let's add this line once we release the stable version of 2.0.21.